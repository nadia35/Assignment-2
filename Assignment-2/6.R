updateZoo <- function(animal_names, animal_rm, animal_add){
  index <- match(animal_rm, animal_names)
  animal_names[index] <- animal_add
  return(animal_names)
}
