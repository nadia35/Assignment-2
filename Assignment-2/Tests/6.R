source("6.R")

mammals = c("rhino", "elephant", "lion", "tiger", "hippo", "cheetah", "panda")
birds = c("parrot", "flamingo", "ostrich", "owl", "emu", "peacock")
fish = c("shark", "piranha")
amphibians = c("frog", "toad", "newt", "salamander", "mantella")

print(updateZoo(mammals, "hippo", "honey badger"))
print(updateZoo(birds, "owl", "falcon"))
print(updateZoo(fish, "piranha", "clownfish"))
print(updateZoo(amphibians, "frog", "mudpuppy"))