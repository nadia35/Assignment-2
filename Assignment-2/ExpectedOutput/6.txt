[1] "rhino"        "elephant"     "lion"         "tiger"        "honey badger"
[6] "cheetah"      "panda"       
[1] "parrot"   "flamingo" "ostrich"  "falcon"   "emu"      "peacock" 
[1] "shark"     "clownfish"
[1] "mudpuppy"   "toad"       "newt"       "salamander" "mantella"  
